static/js/containers/ProfileFormContainer.js

```diff
@@ -177,6 +111,28 @@ class ProfileFormContainer extends React.Component {
     }
   }
 
+  createActionHelper: Function = (actionCreator: Function): (...args: any) => void  => {
+    const { dispatch } = this.props;
+    return (...args) => dispatch(actionCreator(...args));
+  };
```

Comment:
Because dispatch is defined at runtime by the store, so we wouldn't be able to import something.

I think the only alternative to passing it down as a prop would be sticking it on the global object (or on `window` or something).